extensions/functions_comparison.yaml

```diff
@@ -89,51 +143,68 @@ scalar_functions:
     impls:
       - args:
           - value: any1
+            name: x
         return: BOOLEAN
         nullability: DECLARED_OUTPUT
   -
     name: "is_nan"
-    description: Whether a value is not a number.  If input is null, output will be null.
+    description: >
+      Whether a value is not a number.
+
+      If `x` is `null`, `null` is returned.
     impls:
       - args:
           - value: fp32
-        nullability: MIRROR
+            name: x
         return: BOOLEAN
       - args:
           - value: fp64
-        nullability: MIRROR
+            name: x
         return: BOOLEAN
   -
     name: "is_finite"
-    description: Whether a value is finite (neither infinite nor NaN). If input is null, output will be null.
+    description: >
+      Whether a value is finite (neither infinite nor NaN).
+
+      If `x` is `null`, `null` is returned.
     impls:
       - args:
           - value: fp32
-        nullability: MIRROR
+            name: x
+        nullability: DECLARED_OUTPUT
```

Comment:
https://github.com/substrait-io/substrait/pull/303/files#diff-13de9e7b81c09d871b8cc3c28440048686cf37aa76860e44e7155dd3df04e64dL100 (removing `nullability: MIRROR`) is fine because it's the default, but this isn't, because this means that the output will be non-nullable.